virttest/storage.py

```diff
@@ -243,21 +245,22 @@ def cmp(x, y):
     return image_filename
 
 
-def copy_nfs_image(params, image_name, root_dir):
+def copy_nfs_image(params, image_name, root_dir, basename=False):
     """
     copy image from image_path to nfs mount dir if image is not available
     or corrupted.
 
     :param params: Test dict params
     :param image_name: Master image name.
     :param root_dir: Base directory for relative filenames.
+    :param basename: True to use only basename of image_name
     :raise: TestSetupFail if image is unavailable/corrupted
     """
     image_format = params.get("image_format", "qcow2")
     if params.get("setup_local_nfs", "no") == "yes":
         # check for image availability in NFS shared path
         base_dir = params.get("images_base_dir", data_dir.get_data_dir())
-        dst = get_image_filename(params, base_dir)
+        dst = get_image_filename(params, base_dir, basename=basename)
```

Comment:
I guess src can be replaced by, 
```
source = get_image_filename(params, root_dir)
```
instead of this
```
 source = os.path.join(root_dir, "images", image_name)
            if image_format not in source:
                source = "%s.%s" % (source, image_format)
            logging.debug("Checking for image available in image data "
                          "path - %s", source)
```

Reply:
sure, nice catch. I will make the change.